core-rust/state-manager/src/mempool/pending_transaction_result_cache.rs

```diff
@@ -431,14 +454,20 @@ const NON_REJECTION_RECALCULATION_DELAY: Duration = Duration::from_secs(120);
 const MAX_RECALCULATION_DELAY: Duration = Duration::from_secs(1000);
 
 pub struct PendingTransactionResultCache {
+    mempool_deferred_updates_tx: Mutex<Sender<MempoolUpdate>>,
```

Comment:
_(note from author)_

Same note about `Mutex` as with `deferred_updates_rx` from `PriorityMempool`.

Reply:
Same TODO request as in https://github.com/radixdlt/babylon-node/pull/681#discussion_r1330620776 :p